tensor/datafs/list.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2024, Cogent Core. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package datafs
+
+import "strings"
+
+const (
+	Short = false
+	Long  = true
+
+	DirOnly   = false
+	Recursive = true
+)
+
+// todo: list options string
+
+func (d *Data) String() string {
```

Comment:
Is this `String` or `Label`?

Reply:
this is Stringer!  here you can see the diff between `AsString` and `String`.